srcpkgs/wireplumber/template

```diff
@@ -1,22 +1,20 @@
 # Template file for 'wireplumber'
 pkgname=wireplumber
-version=0.4.4
+version=0.4.5
 revision=1
 build_style=meson
 build_helper=gir
 configure_args="-Dintrospection=enabled -Dsystem-lua=true -Dwpipc=enabled"
```

Comment:
according to the changelog of 0.4.2

> - wpipc has been removed and split out to a separate project, https://git.automotivelinux.org/src/pipewire-ic-ipc/

Reply:
Was it ever part of the package? I remember that changelog entry, don't think there was anything to act on.

Reply:
We should at least remove this option, meson 0.60 will make unknown options like this fatal errors

Reply:
I forgot to actually scroll, sorry. Hadn't seen the option.